src/view/app/Query/Update/update.tsx

```diff
@@ -10,186 +10,188 @@ import { Logger } from "../../../../common/Logger";
-                {ProcessAction[action]}
-                </ProBroButton>
-              ) : null 
-              }            
-              <ProBroButton
-                className="button"
-                onClick={() => {
-                  setOpen(false);
-                }}
-              >
-                Cancel
-              </ProBroButton>
-            </div>
-          </div>
-        )}
-      </Popup>
-      <ProBroButton
-        startIcon={<AddIcon />}
-        onClick={insertRecord}
-        disabled={selectedRows.size > 0 ? false : false}
-      >Create</ProBroButton>
-      <ProBroButton
-        startIcon={<EditIcon />}
-        onClick={updateRecord}
-        disabled={selectedRows.size === 1 ? false : true}
-      >Update</ProBroButton>
-      <ProBroButton
-        startIcon={<DeleteIcon />}
-        onClick={deleteRecord}
-        disabled={selectedRows.size > 0 ? false : true}
-      >Delete</ProBroButton>
-    </React.Fragment>
-  );
+    return (
+        <React.Fragment>
+            <Popup open={open} onClose={() => setOpen(false)} modal>
+                {(close) => (
+                    <div className="update-modal">
+                        <div className="update-header">
+                            {tableName}, {ProcessAction[action]}
+                        </div>
+                        <div className="body">
+                            {action === ProcessAction.Delete ? (
+                                <div>
+                                    Are You sure You want delete {selectedRows.size} record
+                                    {selectedRows.size > 1 && "s"}?
+                                </div>
+                            ) : (
+                                <table>
+                                    <tbody>{table}</tbody>
+                                </table>
+                            )}
+                        </div>
+                        <div className="update-btn-container">
+                            {ProcessAction[action] !== "Read" ? (
+                                <ProBroButton className="button" onClick={onSubmitClick}>
+                                    {ProcessAction[action]}
+                                </ProBroButton>
+                            ) : null
+                            }
+                            <ProBroButton
+                                className="button"
+                                onClick={() => {
+                                    setOpen(false);
+                                }}
+                            >
+                                Cancel
+                            </ProBroButton>
+                        </div>
+                    </div>
+                )}
+            </Popup>
+            <ProBroButton
+                startIcon={<AddIcon />}
+                onClick={selectedRows.size === 1 ? copyRecord : insertRecord}
+                disabled={selectedRows.size > 0 ? false : false}
```

Comment:
check this line. I guess icon should be sdisabled then there are more than 1 row selected.